### DISCUSSION TITLE
Download latest fixes as a nuget

### DISCUSSION THREAD

**Discussion by enkaradag** [dotnet/maui#20148] [Q&A]
Hi,

Some of fixes i need are done and addressed to be published for .NET8 SR2.

I saw a comment or @kcrg that says ongoing maui is downloadable as nuget (or perhaps i misunderstood)

https://github.com/dotnet/maui/pull/19931#issuecomment-1905964375

"You can download zip with nugets, put them in some folder and just create local nuget repo in Visual Studio that points to that folder with nugets."

Can i really download and test latest merged "main" as a nuget? If yes, can someone briefly describe how i can "download zip with nugets"? 

I took a look at the discussions about this but could't find anything, sorry if this is a dublicate

Regards

**Comment by kcrg** [comment]
Every pull request even if not merged to main branch is built and u can access azure website with nuget zip to download. I think these nugets only have the changes made in that particular pull request, if u want to test earlier all merged stuff since last service release u can just use nightly feed, more info is in wiki page on GitHub I believe.

In my world, it's about 1 a.m. right now, so I'm already in bed. When the sun comes up, I can record a video for you that will show you where I found this mystical place with nugets.

@davidortinau some more info about that in wiki would be nice. I believe not everyone watch community standups on YouTube with this kind of eastereggs.

**Reply by kcrg** [reply]
@enkaradag 

https://github.com/dotnet/maui/assets/7996711/1b115da5-11ae-4714-8d13-2654957b0857

**Comment by enkaradag** [comment]
Thank you Kacper,

I found "nightly feed" nuget u mentioned in azure website and installed it. For those who is looking for the same thing;

https://dev.azure.com/xamarin/public/_artifacts/feed/maui-nightly

I wasn't aware of that kind of prereleases shared there. You saved my month, i would wait for SR2

Bad news for me, my issue still exists in yesterday's nightly feed. Or perhaps that feed still does not include the fix for my issue tough it is labeled as fixed and closed. I will take a deeper look.

here is what i am waiting for https://github.com/dotnet/maui/pull/20079

Thank you so much

Ender
